pkgs/development/compilers/zig/hook.nix

```diff
@@ -8,6 +8,14 @@ makeSetupHook {
 
   propagatedBuildInputs = [ zig ];
 
+  substitutions = {
+    zig_default_flags =
```

Comment:
Is this passed as one string, not split in whitespaces?
That's strange...

Reply:
this is just plain text substitution with `substituteAll`

Reply:
This is because when I created such arrays the whitespaces were `\`-quoted, generating some nasty bugs.